src/dashboard_client.ts

```diff
@@ -69,6 +69,15 @@ export class LookerEmbedDashboard extends LookerEmbedBase {
     this.send('dashboard:options:set', options)
   }
 
+  /**
+   * Convenience method for opening the dashboard schedule dialog. Only
+   * supported by dashboards next.
+   */
```

Comment:
I've been including the version # that messages are available in the typedocs in lieu of any strict version checking.